plugins/mentions.md

```diff
@@ -93,6 +91,55 @@ tinymce.init({
 });
 ```
 
+### `mentions_item_type`
+
+This option allows you to specify what kind of user interface item to use when displaying the list of users. The `name` item will just display the user's name, while the `profile` item can also display an optional image and description. Depending on what this option is set to, additional properties may be required for the user object provided via [mentions_fetch](#mentions_fetch). Refer to the [User properties](#userproperties) table for more details.
+
+**Type:** `String`
+
+**Default Value:** `name`
+
+**Possible Values:** `name`, `profile`
+
+#### Example: Using `mentions_item_type`
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'mentions',
+  mentions_item_type: 'profile'
+});
+```
+
+##### User properties
+
+What properties are required, optional and not available depends on the [mentions_item_type](#mentions_item_type) setting and the [mentions_select](#mentions_select) setting.
+
+| Name | Value | `name` | `profile` | Description |
+| ---- | ----- | ----------- | ----------- | ----------- |
+| id | string | required | required | Used to identify the user mention in different callbacks |
+| name | string | required | required |  Name to display and highlight search matches on |
+| image | string | not available | optional | Image source for user avatar |
+| description | string | not available | optional | Description to display |
+
+### `mentions_min_chars`
+
+This option makes it possible to specify the number of characters a user need to type after the "@" symbol before the list of users will be displayed.
+
+**Type:** `Number`
+
+**Default Value:** 1
+
+#### Example: Using `mentions_min_chars`
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'mentions',
+  mentions_min_chars: 0
```

Comment:
I'm not sure if it's a deliberate pattern or not, but elsewhere in the docs I'm used to seeing the code snippet for each setting using the default value in the snippet. Feel free to ignore this

Reply:
You're right, fixed!